channels/whatsapp/index.rst

```diff
@@ -0,0 +1,247 @@
+WhatsApp
+========
+
+.. toctree::
+   :hidden:
+
+   preparation
+
+With the release of Zammad 6.3, you can now connect your Zammad with a
+WhatsApp business account, allowing your customers to reach out to you through
+WhatsApp.  🎉
+
+If you already dived deep into this topic, you can head to the setup guide.
+If not, we strongly encourage you to read the prerequisites and limitations
+of this channel carefully and to follow our tutorial for preparing the
+required steps.
+
+We tried to keep it clear and created a separate sub-page for the necessary
+steps on Meta's/WhatsApp's side.
+
+.. warning::
+   Please be aware that the usage of **WhatsApp's cloud API** is mandatory to
+   connect WhatsApp to Zammad. You won't be able
+   to connect to WhatsApp in Zammad if you just use the WhatsApp Business *App*.
+   Have a look
+   `here <https://developers.facebook.com/docs/whatsapp/cloud-api?locale=en_US>`_
+   for further information about the Cloud API.
+
+   There may be costs for you from Meta/WhatsApp side. 1000 service
+   conversations per month are free. However, you should have a look on their
+   `pricing page <https://developers.facebook.com/docs/whatsapp/pricing>`_ to
+   be safe from surprises.
+
+   Be aware that if you go through the whole process, it is very time consuming.
+   You should have some time to follow our preparation guide.
+
+.. _prerequisites:
+
+Prerequisites
+-------------
+
+You can find a guide for the required steps on our
+:doc:`sub-page <preparation>`.
+It covers the following steps:
+
+- Create a Meta business account
+- Create a Meta developer account
+- Create a Meta developer app
+- Add WhatsApp to your app
+- Create a system user
+- Configure your WhatsApp business platform
+- Assign a not yet used phone number
+
+Additionally, you have to meet the following requirements:
+
+- Make sure your system is accessible from the internet because WhatsApp sends
+  messages via webhooks to your system.
+- Make sure to have properly setup your
+  :doc:`FQDN in Settings > System </settings/system/base>` (for the webhook
+  Meta will fire for new messages).
```

Comment:
`for the webhook Meta will fire for new messages` => `for the callback URL Meta will use to deliver new messages`